cfgmgr/intfmgr.cpp

```diff
@@ -122,17 +201,58 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
     SWSS_LOG_ENTER();
 
     string alias(keys[0]);
-    string vrf_name = "";
+    string vlanId;
+    string subIntfAlias;
+    size_t found = alias.find(VLAN_SUB_INTERFACE_SEPARATOR);
+    if (found != string::npos)
+    {
+        // This is a sub interface
+        // subIntfAlias holds the complete sub interface name
+        // while alias becomes the parent interface
+        subIntfAlias = alias;
+        vlanId = alias.substr(found + 1);
+        alias = alias.substr(0, found);
+    }
     bool is_lo = !alias.compare(0, strlen(LOOPBACK_PREFIX), LOOPBACK_PREFIX);
 
+    string vrf_name = "";
+    uint32_t mtu = 0;
+    string adminStatus = "up";
     for (auto idx : data)
     {
         const auto &field = fvField(idx);
         const auto &value = fvValue(idx);
+
         if (field == "vnet_name" || field == "vrf_name")
         {
             vrf_name = value;
         }
+
+        if (field == "mtu")
+        {
+            try
+            {
+                mtu = static_cast<uint32_t>(stoul(value));
+            }
+            catch (const std::invalid_argument &e)
+            {
+                SWSS_LOG_ERROR("Invalid argument %s to %s()", value.c_str(), e.what());
+                continue;
+            }
+            catch (const std::out_of_range &e)
+            {
+                SWSS_LOG_ERROR("Out of range argument %s to %s()", value.c_str(), e.what());
+                continue;
+            }
+
+            // Inherit mtu from parent port or port channel
+            mtu = 0;
```

Comment:
This will overwrite the above assigned `mtu `right? What is the expectation here?

Reply:
At this moment, we do not support user specified mtu. when mtu == 0, the mtu is inherited from the parent port/port channel

Reply:
removed